requirements/requirements.in

```diff
@@ -50,6 +50,7 @@ email_validator==1.1.2
 ethiopian-date-converter==0.1.5
 eulxml==1.1.3
 Faker==5.0.2
+fhirclient~=3.2
```

Comment:
Is this right? It looks like v3 of this library is for `STU-3` FHIR: https://github.com/smart-on-fhir/client-py/#smart-fhir-client

Reply:
Thank you for checking! It seems "4.0.0" is tagged in the repo, but [not yet released to PyPI](https://pypi.org/project/fhirclient/).

If we want to use this library we'll need to install from GitHub.